IBX-11549: `name.help` translation key described in Dev-doc (#3294)
ezsystems/developer-documentation: docs/content_management/pages/page_blocks.md

    For a full example of block configuration, see [Create custom Page block](create_custom_page_block.md).

### Block name help text

The `name` setting accepts either a single translation key, or an array with `text` and `help` keys.
Both `text` and `help` are translatable using the `ibexa_page_fieldtype` translation domain.

> ``` yaml

Reviewer: The YAML example defines `name` twice under the same block, which isn't valid YAML (duplicate mapping key). Suggest splitting it into two snippets:
Scalar form (still supported):
``` yaml
ibexa_fieldtype_page:
    blocks:
        my_block:
            name: my_block.name.key
```
Structured form with a helper text:
``` yaml
ibexa_fieldtype_page:
    blocks:
        my_block:
            name:
                text: my_block.name.key
                help: my_block.name.help.key
```

    For a full example of block configuration, see [Create custom Page block](create_custom_page_block.md).

### Block name help text

The `name` setting accepts either a single translation key, or an array with `text` and `help` keys.
Both `text` and `help` are translatable using the `ibexa_page_fieldtype` translation domain.

> Scalar form (still supported):

Reviewer 2: ```suggestion
Scalar form:
```
`(still supported)` is something we can use when writing the release notes, but here it's not useful.

This sentence might still exist in the doc in 3 years and bring zero value (`still` doesn't mean anything when it's not attached to a version or date)

    For a full example of block configuration, see [Create custom Page block](create_custom_page_block.md).

> ### Block name help text

Reviewer 3: ```suggestion
### Block name and help
```

Author: I'd go with "Block name and help text"

> The `name` setting accepts either a single translation key, or an array with `text` and `help` keys.

Reviewer 3: ```suggestion
The `name` setting accepts either a single translation key, an hard coded string of text that won't be translated, or an object with `text` and `help` property keys.
```

Both `text` and `help` are translatable using the `ibexa_page_fieldtype` translation domain.

Scalar form:

``` yaml
ibexa_fieldtype_page:
    blocks:
        my_block:
            name: my_block.name.key
```

Structured form with a helper text:

```yaml
ibexa_fieldtype_page:
    blocks:
        my_block:
            name:
                text: my_block.name.key
                help: my_block.name.help.key
```

- `text` - corresponds to the block name.

> - `help` - is an optional translation key whose translation is rendered as a helper text under the **Name** field in the block configuration form.

Reviewer 3: Optionally a screenshot to show how it's rendered? Maybe from a built-in block.
